README.md

```diff
@@ -32,3 +32,31 @@ Note that search indexes are only populated for prod build.
 npm run build
 npm run serve
 ```
+
+### Running this on Windows
+
+If you don't already, you might need to set the default shell to `bash`. 
+
+```powershell
+npm config set script-shell "C:\\Program Files\\git\\bin\\bash.exe"
+```
+
+You can revert it later, if you have issues, by doing:
+
+```powershell
+npm config delete script-shell
+```
+
+In some cases, the `postinstall` script doesn't auto-run after `npm install`, in which case, you can run that manually:
```

Comment:
Sounds suspicious, maybe there were some failures? Anyway, doesn't hurt to leave this sentence here.

Reply:
Yeah, it's possible this was a result of me not having `bash` as default for npm. And it failed silently.